.github/workflows/ci.yaml

```
@@ -45,6 +52,7 @@ jobs:
             #   run: composer run-script phpstan
 
             - name: Run code style check
+              if: matrix.code_style_check
```

This change will make it so we don't need to explicitly declare in matrix that we want code style to run. It will run automatically unless we explicitly say otherwise.
```suggestion
              if: !matrix.skip_code_style
```